Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -15,8 +15,8 @@
 
 numWFStart=10000
 numWFSkip=200
-#2017 WFs to run in IB (TenMuE_0_200, TTbar, ZEE, MinBias, TTbar PU, ZEE PU)
-numWFIB = [10021.0,10024.0,10025.0,10026.0,10023.0,10224.0,10225.0]
+#2017 WFs to run in IB (TenMuE_0_200, TTbar, ZEE, MinBias, TTbar PU, ZEE PU, TTbar design)
+numWFIB = [10021.0,10024.0,10025.0,10026.0,10023.0,10224.0,10225.0,10424.0]
```

Comment:
Umm... I need to take a look. Thanks for letting me know. Apparently the workflow has never worked with `-i all`...